main-application/comms-processing_tasks.h

```diff
@@ -0,0 +1,13 @@
+/*  comms-processing_tasks.h
+	Created by Minh Phong Bill Truong on April 28th 2020
+ */
+
+
+//==============================================================================
+//                                    MACROS
+//==============================================================================
+
+
+//==============================================================================
+//                                FUNCTION STUBS
+//==============================================================================
```

Comment:
I usually call these:
- DEFINITIONS (instead of macros)
- PUBLIC GLOBALS
- PUBLIC API STUBS

but that's just my preference lol. If there are no objections I'll likely add these as recommendations to the readme, for the style guide

Reply:
Ack